### PR TITLE
Deprecate upload log from path fix #5117

### LINKED ISSUE #5117: crab uploadlog --logpath= option does not work
--logpath fails due to  missing required parameter:`'X-Error-Info': 'takskname is missing'`

```
[ddirmait@lxplus750 crab_tasks]$ crab uploadlog --logpath=crab.log
Will upload file crab.log.
Fetching user enviroment to log file
Uploading log file...
Fatal error trying to connect to https://cmsweb.cern.ch:8443/crabserver/prod/cache using subresource=upload&objecttype=clientlog. Error details: {'Content-Length': '736', 'X-Error-Http': '400', 'X-Rest-Time': '65973.043 us', 'Server': 'nginx/1.17.10', 'Connection': 'close', 'X-Error-Detail': 'Missing required parameter', 'CMS-Server-Time': 'D=78171 t=1637654377428571', 'X-Rest-Status': '301', 'Date': 'Tue, 23 Nov 2021 07:59:37 GMT', 'Content-Type': 'text/html;charset=utf-8', 'X-Error-Info': 'takskname is missing', 'X-Error-Id': '076b27bf25aa3ce6a6dcd6743618be49'}
ERROR: Exception: Failed to get PreSignedURL from CRAB Server:
url=https://cmsweb.cern.ch:8443/crabserver/prod/cache, code=400, reason=Bad Request, headers={'Content-Length': '736', 'X-Error-Http': '400', 'X-Rest-Time': '65973.043 us', 'Server': 'nginx/1.17.10', 'Connection': 'close', 'X-Error-Detail': 'Missing required parameter', 'CMS-Server-Time': 'D=78171 t=1637654377428571', 'X-Rest-Status': '301', 'Date': 'Tue, 23 Nov 2021 07:59:37 GMT', 'Content-Type': 'text/html;charset=utf-8', 'X-Error-Info': 'takskname is missing', 'X-Error-Id': '076b27bf25aa3ce6a6dcd6743618be49'}

	Please email hn-cms-computing-tools@cern.ch for support with the crab.log file or crab.log URL.
	Client Version: v3.210825
	Please use 'crab uploadlog' to upload the log file /afs/cern.ch/user/d/ddirmait/crab_tasks/crab.log to the CRAB cache.
[ddirmait@lxplus750 crab_tasks]$
```

### PR DESCRIPTION
**Pull request by belforte**

### CODE REVIEW

File: src/python/CRABClient/Commands/uploadlog.py [hunk 1]
```diff
@@ -27,28 +24,25 @@ def __call__(self):
...
             try:
                 taskname = self.cachedinfo['RequestName']
                 logfilename = str(taskname)+".log"
-            except:
-                self.logger.info("Couldn't get information from .requestcache (file likely not created due to submission failure), try\n"
-                        "'crab uploadlog --logpath=<path-to-log-file-in-project-dir>'")
-                return
+            except Exception:
+                self.logger.info("Couldn't get information from .requestcache (file likely not created due to submission failure),\n" +
+                                 "Please local crab.log yourself and copy/paste into the mail to support if needed")
```
Comment (ddaina) [inline src/python/CRABClient/Commands/uploadlog.py:34]: I don't fully understand wording used this message. maybe we should change it to `Please copy/paste crab.log into the mail yourself and send to support if needed.`

**Review by ddaina — APPROVED** [review]
all looks good except message printed to user

File: src/python/CRABClient/Commands/uploadlog.py (hunk 1 above)
Comment (belforte) [reply src/python/CRABClient/Commands/uploadlog.py:34]: oops typo

Comment (belforte) [reply src/python/CRABClient/Commands/uploadlog.py]: please check if message is clear in new version. Thanks

Comment (ddaina) [reply src/python/CRABClient/Commands/uploadlog.py]: looks good, thanks

**Pull request merged**